app/src/main/java/com/codesoom/assignment/service/ToyService.java

```diff
@@ -0,0 +1,74 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.domain.Toy;
+import com.codesoom.assignment.dto.ToySaveRequest;
+import com.codesoom.assignment.repository.ToyRepository;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityNotFoundException;
+
+/**
+ * 장난감 정보에 대한 작업을 처리합니다.
+ */
+@Service
+public class ToyService {
+
+    private ToyRepository toyRepository;
+
+    public ToyService(ToyRepository toyRepository) {
+        this.toyRepository = toyRepository;
+    }
+
+    /**
+     * 장난감 목록을 반환합니다.
+     *
+     * @return 장난감 목록
+     */
+    public Iterable<Toy> list() {
+        return this.toyRepository.findAll();
+    }
+
+    /**
+     * 장난감의 상세정보를 반환합니다.
+     *
+     * @param id 장난감의 id
```

Comment:
`id`가 중복되는 것을 피하고 싶다면 `식별자`라는 표현을 쓰는 방법도 있습니다.
```suggestion
     * @param id 장난감의 식별자
```